test/client/components/NavBar.test.jsx

```diff
@@ -14,4 +20,14 @@ describe("Tests for Nav Bar", () => {
         expect(wrapper.baseElement.outerHTML).toContain("navbar");
     });
 
+    test("Checks Log Out", () => {
+        window.alert = () => {};
+        dataHelper.logOut = mockLogOut
+        Cookies.set('token', 'testtoken123', { expires: 1 });
+        
+        const wrapper = render(<BrowserRouter><NavBar /></BrowserRouter>);
+        fireEvent.click(getByTestId(wrapper.baseElement, "navBarProfilePicture"))
+        fireEvent.click(getByText(wrapper.baseElement, "Log Out"))
+    });
```

Comment:
You should be expecting that the logout function is being called. Check the function mock documentation for how to check that.